src/pug/blog/year_one_recap.pug

```diff
@@ -0,0 +1,67 @@
+    ul
+      li #[em Console Legacy] profile which makes easier to play emulators and retro games.
+      li #[em Racing] profile which allows to use the controllers as a steering wheel.
+      li #[em Generic] gamepad mode (aka DirectInput) implementation, which is useful both for retro gaming and for simulators expecting generic joysticks or steering wheels.
+      li Full #[em Steam Deck] compatibility.
+      li We created the #[em Ctrl app] as an alternative (and optional) method for configuring the Alpakka controller.
+    li There were #[em 5 3D-print models releases] with iterative improvements.
+    li There were #[em 3 PCB releases] with iterative improvements.
+
+  h4 Research
+  ul
+    li A total of #[+a('/db/stats') 44 games evaluated] for the accessibility game database.
+    li Published research notes about #[+a('gyro_ratcheting') Gyro ratcheting and flick-stick].
+    li Done in-depth research about #[+a('alphanumeric_input') Alphanumeric input methods] and implemented some of them in the Alpakka controller.
+
+  hr
+  h3 Things that could have gone better
+  ul
+    li We were too enthusiastic, and we #[em.wrap miscalculated the amount of work 2 individuals can achieve] even if working full-time. We got help from the community with the development, but there are many other aspects of running a company that can easily become a time sink (legal, taxes, insurances, accounting, logistics, support, etc).
+    li The #[em.wrap Kapybara controller development did not start yet], our original plan was to develop it alongside with the Alpakka, but eventually we realized that we do not have the resources to run 2 devices development in parallel, so we decided to delay the Kapybara development until we are done with wireless-related features on the Alpakka.
+    li We onboarded a total of 46 testers in the Testers program, 21 of them with disabilities, but #[em.wrap we never figured out how run the Tester program effectively]. After receiving the initial feedback we were unsuccessful in defining a process to keep the testers engaged, and to keep new feedback and ideas coming. This again boils down to lack of time and resources in our side.
+    li Even though we were successful in making ourselves somewhat visible in the DIY, 3D-printing, and accessibility communities, #[em.wrap we failed to catch the attention of videogame-focused journalism] (eg PC Gamer, IGN, Kotaku, etc). There is not much we can do there other than keep working hard, and sending them emails sometimes.
+    li The #[em wireless / Bluetooth] version of the Alpakka has been much harder to implement than what we expected. We were gladly surprised when a version of the Raspberry Pico with WiFi and Bluetooth was released (Pico-W), and we decided to experiment with it hoping to avoid a lot of trouble. Unfortunately Bluetooth is a extremely complex protocol, and there is still a lot of trouble.
```

Comment:
Rephrase. Something like. "Implementing the wireless version of the Alpakka has proven to be significantly more challenging than anticipated. Especially the wireless communication and the complexity of the Bluetooth protocol required more effort than we though. At the same time, we appreciate the help and support we received from our community."